student.rb

```diff
@@ -1,8 +1,10 @@
 require_relative 'person'
 
 class Student < Person
+  belongs_to :classroom
+
   include Person
-  attr_accessor :classroom
+  attr_accessor :name
```

Comment:
- [ ] As mentioned previously please make sure to add the `attr_reader :classroom` in the Student class as the classroom attribute should be able to be read by entities outside the class.